### PR TITLE
Ruby Project : Finished Version :checkered_flag: :1st_place_medal: 

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### In this branch, We (@HAMMAS-SALEEM, @farouqdaif1, @Lichapa) have worked on the following things :+1: 

:heavy_check_mark: Added Book Class
:heavy_check_mark: Added Label Class
:heavy_check_mark: Added Game Class
:heavy_check_mark: Added Author Class
:heavy_check_mark: Added Music Class
:heavy_check_mark: Added Genre Class
:heavy_check_mark: Added Associations
:heavy_check_mark: Added Encapsulation
:heavy_check_mark: Abstraction
:heavy_check_mark:  Added Inheritance
:heavy_check_mark:  Added Polymorphism
:heavy_check_mark:  Added UI for Application
:heavy_check_mark:  Preserve Data in JSON files
:heavy_check_mark: Added Schema for All Classes
:heavy_check_mark: Followed Gitflow
:heavy_check_mark: Followed Ruby Best Practices
:heavy_check_mark: Followed SOLID
:heavy_check_mark: Followed Linters
:heavy_check_mark: Documented work in professional way

### :+1: :100: :flags: FInally, We have completed the Capstone Project :1st_place_medal: :egypt: :pakistan: :malawi:

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+);
+
+CREATE TABLE BOOKS(
+    ID INT CONSTRAINT ID_BOOKS_PK PRIMARY KEY,
+    PUBLISHER VARCHAR,
+    COVER_STATE VARCHAR,
+    PUBLISH_DATE DATE,
+    ARCHIVED BOOLEAN,
+    LABEL_ID COSTRAINT LABEL-ID_BOOKS_FK REFERENCES LABEL(ID) 
+);
```
Comment (andersonlebon) [inline schema.sql:18]: image

- [ ] Kindly remove the syntax error in this code above (`CONSTRAIN` and  `LABEL_D_BOOKS_FK` also the column type`INT`). Hint :point_down: 

```rb
CREATE TABLE BOOKS(
    ID INT CONSTRAINT ID_BOOKS_PK PRIMARY KEY,
    PUBLISHER VARCHAR,
    COVER_STATE VARCHAR,
    PUBLISH_DATE DATE,
    ARCHIVED BOOLEAN,
    LABEL_ID INT CONSTRAINT LABEL_ID_BOOKS_FK REFERENCES LABEL(ID) 
);
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]

**Review by andersonlebon — APPROVED** [review]
## Hi @team @HAMMAS-SALEEM 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there
- [ ] Highlights:
- Good Pull Request +1

## Required Changes ♻️
_Check the comments above the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (farouqdaif1) [reply schema.sql]: Thanks for feedback we will fix it 😄

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**